qemu/tests/block_copy.py

```diff
@@ -66,15 +65,19 @@ def get_vm(self):
         return live vm object;
         """
         vm = self.env.get_vm(self.params["main_vm"])
-        vm.verify_alive()
+        if self.params.get("start_vm", "yes") == "yes":
+            vm.verify_alive()
         return vm
 
     def get_device(self):
         """
         according configuration get target device ID;
         """
-        image_file = storage.get_image_filename(self.parser_test_args(),
-                                                self.data_dir)
+        params = self.parser_test_args()
+        image_name = params["image_name"]
+        image_format = params["image_format"]
+        image_name += ".%s" % image_format
+        image_file = utils_misc.get_path(self.data_dir, image_name)
```

Comment:
`image_name = "%s.%s" %s (params["image_name"], params["image_format"])` may be more neat.
It does one string formatting instead of one string formatting and string concatenation.